spec/automation/unit/method_validation/filter_by_dialog_parameters_spec.rb

```diff
@@ -49,7 +49,7 @@ def root_service_template_task
       ws = run_automate_method(ServiceTemplate.find_by_name("top"),
                                root_service_template_task,
                                FactoryGirl.create(:service))
-      expect(ws.root['include_service']).to be_true
+      expect(ws.root['include_service']).to be_truthy
```

Comment:
Do you think we should evaluate on a case-by-case basis, or should I reconfigure transpec to use `be true` (if possible) for future conversions?